lib/groups/help.js

```diff
@@ -3,13 +3,18 @@ const chalk = require('chalk');
 const commandLineUsage = require('command-line-usage');
 
 class HelpGroup {
-    outputHelp(command = null) {
-        if (command !== null) {
-            const commands = require('../utils/commands').info;
+    outputHelp(subject = null, isCommand = true) {
+        if (subject !== null) {
+            const info = isCommand ? require('../utils/commands').info : require('../utils/core-flags').info;
+            const options = (() => {
```

Comment:
Why this is and inline IIFE? I don't think we need it.

```js
const options = info.find(commandOrFlag => {
   if (isCommand) {
     // return 
   }
   // return
})
```

Reply:
> Why this is

`options` is the `object` with details about `subject`.

> inline IIFE?

I was recommended to maintain immutability at each step and I didn't had any other implementation in my mind. Thus Opted for inline iffe. Will update it as per your suggestion in next commit.

> I don't think we need it.

Is there any other way to do this?

Reply:
You can use `.reduce` if you really want to keep a functional approach :)